RELEASENOTES-1.4.md

```diff
@@ -341,3 +341,6 @@
 - `release 6 6333`
 - `release 7 7042`
 - `release 7 7041`
+- `note 6` Fixed an issue where initializing a `local` variable via a method call
+  was not properly type-checked, and could result in incorrect semantics,
+  invalid generated C, or undefined behaviour (fixes SIMICS-22874).
```

Comment:
You should explicitly mention the potential effects on existing code: the reader should understand "if I upgrade then I may get a compile error that I didn't see before". You may argue that this can be deduced from your text, but I want something more direct; e.g., I'm happy if a phrase like "will report an error"  is part of the note.

Reply:
Would the following be sufficient?:

> ... was not properly type-checked, which could result in invalid code being accepted, as well as incorrect semantics, ...

Reply:
I've pushed this wording up, protest if you protest.